anchored-keys/src/main/resources/schema/infinispan-anchored-config-11.0.xsd

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" version="1.0"
+           targetNamespace="urn:infinispan:config:anchored:11.0"
+           xmlns:tns="urn:infinispan:config:anchored:11.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+    <xs:element name="anchored-keys" type="tns:anchored-keys"/>
+
+    <xs:complexType name="anchored-keys">
+        <xs:attribute name="enabled" type="xs:boolean" default="2">
```

Comment:
I think we can avoid having the boolean flag. The presence of the `anchored-keys` element should enable the feature. Also `2` is a weird default value for a boolean :)